### ISSUE TITLE
Missing implementation for org.primeframework.mvc.workflow.MVCWorkflow

### ISSUE THREAD

**Issue by Xiakitl** [FusionAuth/charts#40]
### Description 
After deploying a clean installation of FusionAuth with the current Helm Chart, FusionAuth complains about a missing implementation for org.primeframework.mvc.workflow.MVCWorkflow.

_(Istio is installed in the cluster. It doesn't matter if the FusionAuth pods have Istio sidecars injected. Enabling/Disabling the injection of sidecars has no effect on the described error.)_

```
2020-11-13 7:40:58.320 AM ERROR org.primeframework.mvc.servlet.PrimeFilter - Error encountered
com.google.inject.ConfigurationException: Guice configuration errors:

1) No implementation for org.primeframework.mvc.workflow.MVCWorkflow was bound.
  while locating org.primeframework.mvc.workflow.MVCWorkflow

1 error  
	at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1120)
	at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1078)
	at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1131)
	at org.primeframework.mvc.servlet.PrimeFilter.doFilter(PrimeFilter.java:77)
	at com.inversoft.maintenance.servlet.MaintenanceModePrimeFilter.doFilter(MaintenanceModePrimeFilter.java:59)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at com.inversoft.servlet.UTF8Filter.doFilter(UTF8Filter.java:27)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:543)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:615)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:818)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1626)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
```

### Affects versions
- FusionAuth 1.19.8

#### Prepare
1. Create namespace `fusionauth`
`kubectl create namespace fusionauth`
2. (Optional) Enable automatic Istio sidecar injection
`kubectl label namespace fusionauth istio-injection=enabled`
3. Install Postgresql in namespace `fusionauth` and ElasticSearch in namespace `elastic`
4. Enable Authentication in Elastic
5. Configure Elasticsearch users

#### Reproduce
1. Install FusionAuth
```
helm install fusionauth fusionauth/fusionauth
    --set database.root.user=postgres
    --set database.root.password=<Hidden>
    --set database.user=postgres
    --set database.password=<Hidden>
    --set database.host=postgresql-postgresql-0.postgresql-headless.fusionauth.svc.cluster.local
    --set search.host=elasticsearch-master-0.elasticsearch-master-headless.elastic.svc.cluster.local
    --set search.user=elastic
    --set search.password=<Hidden>
    --namespace fusionauth
```
2. Look into the logs of `elasticsearch` container

### Platform
 - OS: Docker and Kubernetes running on Ubuntu 18.04 LTS

### Additional context
The InitContainers succeed. Both, ElasticSeach and Postgresql, are reachable from FusionAuth.

Removing all Helm Charts (ES, Postgres, FusionAuth) incl. removing the PVCs and reinstalling doesn't change anything.

**Comment by robotdan** [comment]
This seems to be that you are missing jars in your class path. What FusionAuth image are you using?

**Comment by Xiakitl** [comment]
I didn't change the value of the image or tag in the Chart. Therefore the fusionauth/fusionauth-app:1.19.8 is used.

**Comment by Xiakitl** [comment]
I changed the tag via `--set image.tag="1.21.0"`. It works now.
It seems that 1.19.8 is broken. Maybe this Chart should be updated, to use a newer non-broken image.

**Comment by robotdan** [comment]
Thanks for the update @Xiakitl  - strange, that 1.19.8 isn't working. 

I was able to start up FusionAuth `1.19.8` ok using `docker run -p 9011:9011 fusionauth/fusionauth-app:1.19.8`. 

Can you try deleting `1.19.8` image locally and re-trying? Perhaps your local image is corrupted or something (I don't know if this is even possible).

**Comment by drpebcak** [comment]
I've just released a new version of the chart that uses 1.21.0 instead. If there are other issues with the 1.19.8 image, this issue should probably be moved elsewhere (http://github.com/fusionauth/fusionauth-containers) since it doesn't have anything to do with the chart.
